src/Tokenizers/PHP.php

```diff
@@ -1727,7 +1727,7 @@ protected function tokenize($string)
                     || (stripos($newContent, '0b') === 0 && bindec(str_replace('_', '', $newContent)) > PHP_INT_MAX)
                     || (stripos($newContent, '0o') === 0 && octdec(str_replace('_', '', $newContent)) > PHP_INT_MAX)
                     || (stripos($newContent, '0x') !== 0
-                    && stripos($newContent, 'e') !== false || strpos($newContent, '.') !== false)
+                    && (stripos($newContent, 'e') !== false || strpos($newContent, '.') !== false))
```

Comment:
Unless it's too early in the morning and I misread this, it even caught a bug in here 🙌

Reply:
I thought so too when I made the change, but there is a test for this code and it passed before and after 🤷🏻‍♀️